src/layouts/EditPage/EditPage.tsx

```diff
@@ -125,6 +126,7 @@ export const EditPage = () => {
       TableCell,
       Underline,
       Placeholder,
+      TrailingNode,
```

Comment:
out of scope, but would it be possible for you to extract all these into a `useTiptapEditor` custom hook for organisation purposes?

Reply:
eh nvm i'll do this!